zerver/openapi/zulip.yaml

```diff
@@ -12884,9 +12896,13 @@ paths:
                               - 2 - Private messages and mentions
                               - 3 - None
                           realm_name_in_notifications:
-                            type: boolean
+                            type: integer
                             description: |
                               Include organization name in subject of message notification emails.
+
+                              - 1 - Automatic
```

Comment:
Would be better to have a short description of these options here also, especially for `Automatic`.

Reply:
I have updated the description to make it consistent with [this](https://github.com/zulip/zulip/blob/main/zerver/openapi/zulip.yaml#L13801).